src/core/utils/src/object.js

```diff
@@ -26,3 +30,18 @@ export function nested(obj, dotProperty, value) {
 
   return value || obj;
 }
+
+
+/**
+ * @private
+ */
+export function keyBy(array, iteratee) {
```

Comment:
Are we trying to remove lodash as a dependency? I saw avoiding its functions and this is a function it has, being added.

Reply:
Yes we are going to remove [lodash](https://lodash.com/) as a dependency in the near future. Some functions we might still need so we will add them to the appropriate place in the utils folder. I don't foresee this happening much because most of the functions we are using from [lodash](https://lodash.com/) are validation functions.